lib/msf/core/rpc/v10/job_state_tracker.rb

```diff
@@ -0,0 +1,62 @@
+class JobStateTracker
+
```

Comment:
iirc we'll want to include the MonitorMixin here too

https://ruby-doc.org/stdlib-2.6.1/libdoc/monitor/rdoc/MonitorMixin.html#module-MonitorMixin-label-Simple+Class+include

Reply:
yup, 100% will do